napari/plugins/__init__.py

```diff
@@ -64,3 +65,20 @@ def _initialize_plugins():
         plugin_manager._initialize()
 
     _install_npe2_themes()
+
+    # FIXME is this the right place to initialize plugin keybindings?
+    from napari._app_model import get_app
+    from napari.utils.shortcuts import plugins_shortcuts
+
+    shortcut_settings = get_settings().shortcuts.shortcuts
+    for command, keybinding_rules in plugins_shortcuts.items():
+        if command in shortcut_settings:
```

Comment:
It looks like we're also initializing shortcuts from settings here which seems kind of odd? I feel like they should be initialized elsewhere, and override plugin shortcuts if they exist

Reply:
For context, this is just a temporary workaround that will be fixed when we transition over to the new system completely.